python/api/src/araali/__main__.py

```diff
@@ -510,6 +512,77 @@ def fw_config(args):
+        EKS_STACK_NAME = "araali-quickstart-stack-%s"%timestamp_str
+        EKS_CLUSTER_NAME = "araali-quickstart-eks-%s"%timestamp_str
+        cluster_name = args.name
+        launch_cluster = True
+        if cluster_name == "":
+            launch_cluster = True
+            cluster_name = EKS_CLUSTER_NAME
+        else:
+            # Check if cluster exists
+            response = eks_client.list_clusters()
+            if "clusters" not in response:
+                print("Error: list_clusters returned invalid response")  
+                sys.exit(1)  
+            launch_cluster = cluster_name not in response["clusters"]
+        # Launch cluster is requested
+        if launch_cluster:
+            success, result = _aws.cf_launch_eks_cluster(stack_name=EKS_STACK_NAME, cluster_name=cluster_name, availability_zones=args.availabilityzones)
+            if not success:
+                print("Error: Quickstart EKS cluster setup failed")
+                sys.exit(1)
+            # Wait until cluster is set up
+            counter = 0
+            while(True):
+                if counter % 10 == 0:
+                    print("Waiting for cluster create - %d/120" %counter)
+                if _aws.cf_validate_stack_creation(EKS_STACK_NAME):
+                    break
+                if counter >= 120:
+                    print("Error: Quickstart EKS cluster setup failed")
+                    sys.exit(1)
+                counter += 1
+                time.sleep(30)
+        # Update kubeconfig to point to newly created cluster and fetch ARN
+        utils.update_kubeconfig(cluster_name)
+        response = eks_client.describe_cluster(name=cluster_name)
+        if "cluster" not in response and "arn" not in response["cluster"]:
+            sys.exit(1)
+        cluster_arn = response["cluster"]["arn"]
+        # Install Helm on machine if not present
+        # Verify if Helm is present on machine
+        rc = utils.run_command("which helm", debug=False, result=True, strip=False)
+        if rc[0]:
+            # Helm is not present. Download Helm on machine
+            success, _ = utils.run_command_logfailure("curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3")
+            if not success:
+                sys.exit(1)
+            success, _ = utils.run_command_logfailure("chmod 700 get_helm.sh")
+            if not success:
+                sys.exit(1)
+            success, _ = utils.run_command_logfailure("./get_helm.sh")
+            if not success:
+                sys.exit(1)
+        # Run helm install command
+        success, _ = utils.run_command_logfailure("helm repo add araali-helm https://araalinetworks.github.io/araali-helm/")
+        if not success:
+            sys.exit(1)
+        success, _ = utils.run_command_logfailure("helm install araali-quickstart-nanny --set email=%s araali-helm/araali-quickstartnanny --kube-context=%s"%(args.email,cluster_arn))
```

Comment:
you havent addressed this either. kubeconfig is not an atomic operation. You need to use the right parameter in helm to make sure you are referring to the right cluster.

Reply:
I am doing that. Have added --kube-context in the Helm install command.